wix-secrets-backend/wix-secrets-backend.service.json

```diff
@@ -1,41 +1,153 @@
 { "name": "wix-secrets-backend",
   "mixes": [],
   "labels":
-    [ "new" ],
+    [ "changed" ],
   "location":
     { "lineno": 1,
-      "filename": "secrets.js" },
+      "filename": "getSecret.js" },
   "docs":
-    { "summary": "The wix-secrets-backend module contains functionality for\nretrieving secrets you safely store in your site's <a href=\"https://support.wix.com/en/article/corvid-about-the-secrets-manager\" target=\"_blank\">Secrets Manager</a>.",
+    { "summary": "The wix-secrets-backend module contains functionality for\nmanaging secrets you safely store in your site's [Secrets Manager](https://support.wix.com/en/article/corvid-about-the-secrets-manager).",
       "description":
         [ "The Secrets Manager allows you to store and encrypt API keys and other secrets on your site.",
           "With the Secrets API, you can safely work with secrets in your backend code.",
           "",
           " Follow this general procedure for working with API keys or other secrets using the Secrets Manager:",
           "",
           " 1. Get private information such as an API key from a 3rd-party service.",
-          " 1. Store the private information as a new secret in the <a href=\"https://support.wix.com/en/article/corvid-working-with-the-secrets-manager\" target=\"_blank\">Secrets Manager</a> in your site's dashboard. Assign a name to the secret.",
+          " 1. Store the private information as a new secret in the [Secrets Manager](https://support.wix.com/en/article/corvid-working-with-the-secrets-manager) in your site's dashboard or with the `createSecret()` function. Assign a name to the secret.",
           " 1. In your backend code, instead of hardcoding the API key, use the `getSecret()` function with the secret name assigned in the Secrets Manager. When the code runs, the value of the secret is extracted from the Secrets Manager.",
           "",
           "",
-          "**Note:** By default, site contributors do not have permission to create secrets",
+          "> **Notes:**",
+          "> + By default, site contributors do not have permission to create secrets",
           "and view secret values in the Secrets Manager. Contributors who have access to the",
           "editor can retrieve a secret's value using code.",
+          ">",
+          ">",
+          "> + To use the Secrets API, you must add a Members Area to your site.",
```

Comment:
Maybe a link to https://support.wix.com/en/article/adding-a-members-area-to-your-site?

Reply:
Good thinking